wgpu-hal/Cargo.toml

```diff
@@ -134,7 +134,7 @@ features = ["wgsl-in"]
 [dev-dependencies]
 cfg-if = "1"
 env_logger = "0.10"
-winit = "0.27.1"   # for "halmark" example
+winit.workspace = true   # for "halmark" example
```

Comment:
We can't workspace dependencies in any crate depended on by Firefox due to a (now fixed, but yet to be adopted) bug in cargo's vendoring.

Reply:
See https://github.com/gfx-rs/wgpu/issues/3407 for more info